### PR TITLE
[nodes] calibration nodes

### PR DESCRIPTION
**Pull request by servantftechnicolor**
Add calibration nodes :

- 1 node for checkerboard detection
-  1 node for distortion estimation from checkerboards
- 1 node for intrinsics estimation from checkerboards

### CODE REVIEW

**Comment by natowi** [comment]
It would be great to add support for the CameraInit node to use the calibration results as input :)

**Comment by servantftechnicolor** [comment]
> Can the sfmData.sfm now be used as input for the CameraInit node? If not, that would be great :)

Currently we can use sfmtransfer to copy intrinsics to the output of camera init. However, it may be not user friendly?

**Comment by natowi** [comment]
Using the input in the CameraInit node would be more intuitive I think: grafik

File: meshroom/nodes/aliceVision/RigCreate.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+from meshroom.core import desc
+
+
+class RigCreate(desc.CommandLineNode):
+    commandLine = 'aliceVision_utils_rigCreate {allParams}'
+    category = 'Other'
+    documentation = '''
+Create a rig from multiple input sfm files.
+No modification is done on the data. Rig calibration is not performed here.
```
Comment (natowi) [inline meshroom/nodes/aliceVision/RigCreate.py:14]: Better: Rig calibration can be performed with...

**Pull request closed without merging**